packages/fxa-auth-server/lib/routes/totp.js

```diff
@@ -130,6 +131,16 @@ module.exports = (log, db, mailer, customs, config, glean, profileClient) => {
         auth: {
           strategy: 'sessionToken',
         },
+        validate: {
+          payload: isA.object({
+            code: isA
+              .string()
+              .max(32)
```

Comment:
sorry I have not been following the SMS work that closely but is this a code sent to the user?

Reply:
Yes it would be a code sent to the users phone. There's some things about this ticket that are a little fuzzy to me as well. I'm thinking when we do the front end implementation the usage will be more clear.

Reply:
32 characters is a looooong code.  😅